### ISSUE TITLE
Error when running rosehip in the interpreter

### ISSUE THREAD

**Issue by nehol** [donno2048/CITUR-L#7]
![image](https://user-images.githubusercontent.com/70110929/91045468-9a943400-e61f-11ea-840a-4300aa430560.png)

**Comment by donno2048** [comment]
I updated Rosehip so you could just use:
```python
from Rosehip import start
start()
```
This problem looks to have something to do with this [post](https://stackoverflow.com/questions/338768)

**Comment by nehol** [comment]
It's still not working and btw it has nothing to do with this post since in ubuntu it's working and in rosehip not

**Comment by donno2048** [comment]
If you can import _Rosehip_ but not _Rosehip.os_ it's because the _\_\_init\_\_.py_ is not being recognized for some reason

**Comment by donno2048** [comment]
It might be fixed using:
```bash
mv build/external/overlay/usr/share/Rosehip/startup.py build/external/overlay/usr/share
yes | rm -r build/external/overlay/usr/share/Rosehip
cp -a Rosehip/. build/external/overlay/usr/share/Rosehip
mv build/external/overlay/usr/share/startup.py build/external/overlay/usr/share/Rosehip
```
before the two last lines of _init.sh_; and then in _startup.py_ replace:
```python
from Rosehip.os import OS;return OS().run()
```
with:
```python
from Rosehip import pc;pc()

**Comment by nehol** [comment]
The build through the test branch is not working either

**Comment by donno2048** [comment]
I might have to add the [rosehip pip module](https://pypi.org/project/rosehip/) manually like the other modules I will try to reopen the test branch like that

**Comment by donno2048** [comment]
It won't work as long as the pip module is still buggy

**Comment by donno2048** [comment]
No need for pip installation, the problem is when cloning the Rosehip folder will be empty

**Comment by donno2048** [comment]
Fixed it in #12
